www-apps/radicale/radicale-3.0.6-r2.ebuild

```
@@ -23,50 +23,60 @@ RDEPEND="
 	acct-group/radicale
 	dev-python/defusedxml
 	dev-python/passlib[bcrypt,${PYTHON_USEDEP}]
-	>=dev-python/vobject-0.9.6[${PYTHON_USEDEP}]
-	>=dev-python/python-dateutil-2.8.1[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/vobject[${PYTHON_USEDEP}]
+	dev-python/waitress[${PYTHON_USEDEP}]
 	sys-apps/util-linux
 "
 
+BDEPEND="test? ( ${RDEPEND} )"
+
 S="${WORKDIR}/${MY_P}"
 
-RDIR=/var/lib/${PN}
+RDIR=/var/lib/"${PN}"
 
-python_install_all() {
-	rm README* || die
+DOCS=( DOCUMENTATION.md NEWS.md README.md )
```

`README.md` doesn't seem to contain any useful info.

```suggestion
DOCS=( DOCUMENTATION.md NEWS.md )
```

---

Fixed